src/server/routes/login.js

```diff
@@ -15,16 +15,16 @@ const router = express.Router();
 
 /**
  * Authenticate users and return a JSON Web Token with their user ID.
- * @param {String} email
+ * @param {String} username
  * @param {String} Password
  */
 router.post('/', async (req, res) => {
 	const validParams = {
 		type: 'object',
 		maxProperties: 2,
-		required: ['email', 'password'],
+		required: ['username', 'password'],
 		properties: {
-			email: {
+			username: {
 				type: 'string',
 				minLength: 3,
```

Comment:
Having a minimum and maximum length could be a good idea for security reasons. 
I think matching them here and in the creation modal would be a good idea.

Reply:
I will remove the minLength as requested as your explanation makes sense to me.  I will add 128 as max password, to prevent bad actors mucking with it.

Reply:
let me know if you think 128 is not large enough.

Reply:
I think we should put back minLength but set it to 8 on the password. Username can be any min length.

Reply:
OK I will add back minLength to 8 on password and 3 on username.